tests/installation/bootloader_settings/disable_boot_menu_timeout_right_key.pm

```diff
@@ -0,0 +1,19 @@
+# SUSE's openQA tests
+#
+# Copyright SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: Disable grub timeout from the Installer
```

Comment:
I don't think nobody would understand what you mean with `_right_key` name, I would replace everywhere by `_navigating_tabs`. Do you want to test it with someone? wdyt?